internal/pkg/cli/validate.go

```diff
@@ -178,6 +205,37 @@ func basicNameValidation(val interface{}) error {
 	return nil
 }
 
+func basicCronValidation(sched string) error {
```

Comment:
nit: can we remove the "basic" prefix as it doesn't add additional clarifications?
```go
func validateSchedule(sched string) error
func validateDuration(duration string) error
```

Reply:
I'll change `basicCronValidation` to `validateCron` so it doesn't overlap with the new `validateSchedule` (formerly `validateJobSchedule`), below, and so it's more accurate, too. (duration and cron at one level; timeout and schedule at another)